src/ModelerApp.vue

```diff
@@ -90,6 +90,9 @@ export default {
   created() {
     reader.onload = this.setUploadedXml;
   },
+  mounted() {
+    window.ProcessMaker.$modeler = this.$refs.modeler;
```

Comment:
Fixed. I replaced `ProcessMaker.$modeler.definitions.rootElements` -> `store.getters.rootElements`

Reply:
I could not remove 
```
     window.ProcessMaker.$modeler = this.$refs.modeler;
```
Because I still need a reference to `moddle` to create new signals from inside the SignalSelect component.

`window.ProcessMaker.$modeler.moddle.create()`